states/texas/elections.toml

```diff
@@ -1,13 +1,52 @@
-[20200303]
-date = 2020-03-03
-name = "Congressional & Presidential Primary"
-original_date = 2020-03-03
-
 [20200714]
 date = 2020-07-14
 name = "Congressional Primary Run-Off (if needed)"
 original_date = 2020-07-14
 
+[20200714.registration]
+received_by = 2020-06-15
+postmarked_by = 2020-06-15
+in_person_by = 2020-06-15
+sources = ["https://www.sos.state.tx.us/elections/voter/important-election-dates.shtml",
+            "https://www.votetexas.gov/register-to-vote/you-must-register-by.html",
+            "https://www.votetexas.gov/faq/index.html#2798"]
+
+[20200714.poll]
+in_person_starts = 2020-07-14T07:00:00
+in_person_by = 2020-07-14T19:00:00
+sources = ["https://www.sos.state.tx.us/elections/voter/important-election-dates.shtml"]
+
+[20200714.poll.early]
+in_person_starts = 2020-06-29
+in_person_by = 2020-07-10
+sources = ["https://www.sos.state.tx.us/elections/voter/important-election-dates.shtml",
+           "https://www.votetexas.gov/faq/early-voting.html#2773"]
+
+[20200714.absentee]
+received_by = 2020-07-15T17:00:00
+postmarked_by = 2020-07-14T19:00:00
+sources = ["https://www.sos.state.tx.us/elections/voter/important-election-dates.shtml",
+           "https://www.votetexas.gov/military-overseas-voters/index.html",
+           "https://www.sos.texas.gov/elections/voter/reqabbm.shtml"]
+
+[20200714.absentee.overseas]
+received_by = 2020-07-21
```

Comment:
From sources:
"Deadline for voters voting from overseas location: receipt by 5th day after Election Day"
I would make this 2020-07-19. I know that's a Sunday but they don't call out "business days".

Reply:
I was wondering about this when counting. Forgot to bring it up when I pushed. I'll make the change!

Reply:
```suggestion
received_by = 2020-07-19
```